src/fragments/lib/auth/js/mfa.mdx

```diff
@@ -109,46 +212,144 @@ in the current session and set it using `user.sendMFASelectionAnswer` function.
 
 The following code is only for demonstration purpose:
 
-```javascript
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { CognitoUser } from '@aws-amplify/auth';
```

Comment:
we are trying to avoid importing `CognitoUser` from the scope package as this pattern will disappear on the next major version of the library.

```suggestion
```